src/Tools/dotnet-user-secrets/test/InitCommandTest.cs

```diff
@@ -61,6 +61,7 @@ public void AddsSpecificSecretIdToProject()
         }
 
         [Fact]
+        [Flaky("<no longer needed; tracked in Kusto>", FlakyOn.All)]
```

Comment:
![image](https://user-images.githubusercontent.com/7574801/74357544-ee6eb800-4d74-11ea-822d-f31f564a28b5.png)


Looks like you could use a `using`